cmd/airgap/airgap.go

```diff
@@ -19,16 +19,24 @@ package airgap
 import (
 	"github.com/k0sproject/k0s/pkg/config"
 
+	"github.com/sirupsen/logrus"
 	"github.com/spf13/cobra"
 	"github.com/spf13/pflag"
 )
 
 func NewAirgapCmd() *cobra.Command {
 	cmd := &cobra.Command{
 		Use:   "airgap",
-		Short: "Manage airgap setup",
-		Args:  cobra.NoArgs,
-		RunE:  func(*cobra.Command, []string) error { return pflag.ErrHelp }, // Enforce arg validation
+		Short: "Tooling for airgapped installations",
+		Long: `Tooling for airgapped installations.
+
+For example, to create an image bundle that contains the images required for
+the current configuration, use the following command:
+
+    k0s airgap list-images | k0s airgap bundle-artifacts -v -o image-bundle.tar
```

Comment:
Does it make sense to run only `k0s airgap list-images | k0s airgap bundle-artifacts -v -o image-bundle.tar` for the default images?

Reply:
I think the idea having the input is for the users to be able to add their own images too. But that said, it _could_ work also in the way that if the list is not given, then it could use the default images. But would that need config loading etc.?

Reply:
Yeah, that's what I mean, if the input list is not provided, use default images. If I remember correctly, existing k0s airgap list-images reads config already, so the user could use pipes.

Reply:
Yes, that was kinda the idea: Not to add any config loading here at all and use pipes. Otherwise this would be duplicating quite a bit of the logic in `list-images`. This is also advertised in the long help text of `k0s airgap`:

> For example, to create an image bundle that contains the images required for
> the current configuration, use the following command:
>
>    k0s airgap list-images | k0s airgap bundle-artifacts -v -o image-bundle.tar